backend/README.md

```diff
@@ -0,0 +1,83 @@
+## Setup 
+
+This is a step by step guide to setup the MealCare Backend for Linux/MacOS
+
+### Requirements
+
+
+Python 3.6+, Pip 
+
+### Virtual Environment
+
+Start up your Terminal (Unix) or Powershell, and `cd` into `MealCare/backend`
+
+Create a virtual environment for this project using `venv` in Python 3:
+
+```zsh
+python3 -m venv env
+```
+Note: The environment name "env" can be changed to whatever you want. Just make sure to not add it in your commits.
+
+Start the virtual environment using:
+```zsh
+source env/bin/activate
+```
+
+To close the environment use `deactivate`
+
+
+
+The list of all the necessary dependencies can be found in the requirements.txt. 
+To install all of the necessary dependencies in it use:
+
+```text
+pip install -r path/to/requirements.txt
+```
+
+When finished check your dependencies with:
+
+```text
+pip list
+```
+
+### Running FastAPI
+
+
+To run FastAPI locally either run the server using uvicorn (ASGI Server) on your terminal with:
+```
+uvicorn manage:app --reload
+```
+
+Or using:
+
+```text
+python3 manager.py
+```
+
+Or using a debugger in Pycharm or VSCode
+
+To change the mode, create a file named `.env`:
+```text
+FAST_API_CONFIG=value
+```
+`value` can be either `development`, `production`, `testing`. If not defined it will fallback to `development`.
+
+Once ran, check `http://127.0.0.1:8000`.
+
+It should display MealCare. 
+
+Congrats you got FastAPI is running locally!
+
+### Postgres
+
+To install Postgres based on your os follow this guide:
+https://wiki.postgresql.org/wiki/Detailed_installation_guides
+
+Setup the Database and Admin by running this script:
+```zsh
+python3 ./scripts/setup_db.py
```

Comment:
Great work Philip! One small thing, I would move the postgres setup before the uvicorn setup. An error is thrown for the app if it cannot connect to the postgres database (which is good) since it is not setup yet.

Reply:
Good point I'll do that!